src/OutputFormatter/ConsoleOutputFormatter.php

```diff
@@ -75,7 +72,7 @@ private function printViolation(Rule $rule, Output $output): void
                 $rule->getDependentLayer()
             )
         );
-        $this->printFileOccurrence($output, $dependency->getFileOccurrence());
+        $this->printFileOccurrence($output, $dependency->getFileOccurrence()->getFilepath(), $dependency->getFileOccurrence()->getLine());
```

Comment:
Why this change? Previously the private method contained the logic for how the occurrence is rendered and now that detail leaks out into the callee. As a result, whenever we use the private method we need to "disassemble" the occurrence manually. If we ever change the signature we will now have to adjust more places increasing the risk of errors.

Admittedly, this is something that phpstan will likely catch, but for me the previous version feels "better".

Reply:
This was a part of the cheeky commit: "OutputFormatter and Ast (this one is a silly hack since deptrac does not recognize transitive dependencies yet, if it did, this refactor would not fly)" The actual problem is transitive dependency on internal implementation that will need a more in-depth solution. I am fine to revert this one and instead mark it in deptrac baseline